doc/man/scons.xml

```diff
@@ -102,35 +102,49 @@ orchestrates the construction of software
 (and other tangible products such as documentation files)
 by determining which
 component pieces must be built or rebuilt and invoking the necessary
-commands to build them.</para>
+commands to build them.
+&SCons; offers many features to improve developer productivity
+such as parallel builds, caching of build artefacts, and
+a database of information about previous builds so details do
+not have to be recalculated each run.
+</para>
 
-<para>You instruct
-&scons; by writing a configuration file
-which specifies the files to be built (<firstterm>targets</firstterm>),
-and, if necessary, the rules to build those files.  Premade
-rules exist for building many common software components
+<para>You set up an &SCons;
+build system by writing a configuration file
+that describes things to build (<firstterm>targets</firstterm>), and,
+if necessary, the rules to build those files (<firstterm>actions</firstterm>).
+&SCons; comes with a collection of <firstterm>Builder</firstterm> methods
+which apply premade rules for building many common software components
 such as executable programs, object files and libraries,
 so that for many software projects,
-only the target and input files (<firstterm>sources</firstterm>)
-need be specified.</para>
+only the targets and input files (<firstterm>sources</firstterm>)
+need be specified in a call to a builder.
+&scons; thus can operate at a level of abstraction above that of pure files.
+For example if you specify a library target named "foo",
+&scons; keeps track of the actual operating system dependent filename
+(for example <filename>libfoo.so</filename> on a GNU/Linux system),
+and how to refer to that library in later construction steps
+that want to use it, so you don't have to specify that precise
+information yourself.
+&scons; can also scan automatically for dependency information,
+such as header files included by source code files,
+so this does not have to be specified manually.
+</para>
 
 <para>
 When invoked, &scons;
-searches for a file named
+looks for a file named
 &SConstruct;
-(it also checks alternate spellings
-&Sconstruct;, &sconstruct;, &SConstruct.py;, &Sconstruct.py;
-and &sconstruct.py;
-in that order) in the current directory and reads its
-configuration from that file.
-An alternate file name may be
-specified via the
-<option>-f</option>
-option.
+in the current directory and reads the
+build configuration from that file
+(other names are possible,
+see <xref linkend="configuration_files"/> for more information).
+An &SConstruct; file is required:
```

Comment:
A SConstruct ?

Reply:
as previous - unless you want to prefer the consontant-sound pronouniation - I'll have to make a few other changes if so.